docs/en/siem/detection-engine-intro.asciidoc

```diff
@@ -77,25 +77,50 @@ To investigate a signal in the Timeline, click the *View in timeline* icon.
 [[detections-permissions]]
 === {kib} and index privileges required for Detections
 
-[IMPORTANT]
+Depending on your privileges and whether a `.siem-signals-<space name>` index 
+has been created for the {kib} space, you might see an error message when you 
+try to open the *Detections* page.
+
+[NOTE]
 ==============
-You cannot view the Detections page until a user with the `create_index` 
-privilege for the {kib} space visits the page.
+If you are using a local {es} deployment and cannot access the {siem-app}, make 
+sure the `xpack.security.enabled` setting in the `elasticsearch.yml` 
+configuration file is set to `true`. For more information, see 
+{ref}/settings.html[Configuring {es}] and
+{ref}/security-settings.html[Security settings in {es}].
 ==============
 
+[float]
+==== `Let’s set up your detection engine` message
+
+If you see this message, a user with these privileges must open the *Detections*
+page before you can view the page:
```

Comment:
> before you can view the page:

nit: Consider changing `before you can view the page` to something like `before you can enable detection rules and view signals`, because technically the user can always view the page; it just displays a message instead of showing the usual contents.